app/src/main/java/com/example/seonoh/seonohapp/TabPagerAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.seonoh.seonohapp
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import java.lang.NullPointerException
+
+class TabPagerAdapter : FragmentStatePagerAdapter {
+
+    private var tabCount = 0
+
+    constructor(fm: FragmentManager, tabcount: Int) : super(fm) {
```

Comment:
```suggestion
   class TabPagerAdapter(fm : FragmentManager, private val tabCount:Int) : FragmentStatePagerAdapter (fm){
```
부생성자 필요없이 주생성자에서 처리하면 tabCount도 immutable하게 설정해서  사용할 수 있을것 같습니다.

Reply:
네 확인했습니다!